### ISSUE TITLE
[Thumbnail] Unity

### ISSUE THREAD

**Issue by ModProg** [andOTP/andOTP#624]
Please fill in **all the information below** to help speed up the process!

**Name**: Unity (id.unity.com)
**Website**: https://unity.com 
**Link to SVG or EPS image**:  https://unity.com/sites/default/files/unity-logo-assets.zip

```
<svg
   xmlns:dc="http://purl.org/dc/elements/1.1/"
   xmlns:cc="http://creativecommons.org/ns#"
   xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
   xmlns:svg="http://www.w3.org/2000/svg"
   xmlns="http://www.w3.org/2000/svg"
   viewBox="0 0 546.06665 289.98666"
   height="289.98666"
   width="546.06665"
   xml:space="preserve"
   id="svg2"
   version="1.1"><metadata
     id="metadata8"><rdf:RDF><cc:Work
         rdf:about=""><dc:format>image/svg+xml</dc:format><dc:type
           rdf:resource="http://purl.org/dc/dcmitype/StillImage" /><dc:title></dc:title></cc:Work></rdf:RDF></metadata><defs
     id="defs6" /><g
     transform="matrix(1.3333333,0,0,-1.3333333,0,289.98667)"
     id="g10"><g
       transform="scale(0.1)"
       id="g12"><path
         id="path14"
         style="fill:#100f0d;fill-opacity:1;fill-rule:nonzero;stroke:none"
         d="m 1515.41,1630.35 -443.51,-115.57 -65.63,-112.64 -133.204,0.97 -324.636,-315.72 324.636,-315.652 133.134,0.973 65.78,-112.57 443.43,-115.571 118.8,431.149 -67.5,111.671 67.5,111.67 z m -498.46,-246.34 339.26,84.8 -194.75,-328.08 H 771.91 Z m 0,-593.158 -245.04,243.268 h 389.55 l 194.75,-328.069 z m 434.2,-31.473 -194.82,328.011 194.82,328.15 94.07,-328.15 -94.07,-328.011" /><path
         id="path16"
         style="fill:#100f0d;fill-opacity:1;fill-rule:nonzero;stroke:none"
         d="m 2521.41,1310.45 c -59.27,0 -99.05,-24.56 -128.69,-71.11 h -2.54 v 59.26 H 2286.46 V 864.289 h 106.26 v 246.371 c 0,59.26 37.25,99.89 88.05,99.89 48.25,0 84.22,-28.61 84.22,-79.58 V 864.289 h 106.26 V 1147.9 c 0,94.83 -60.11,162.55 -149.84,162.55" /><path
         id="path18"
         style="fill:#100f0d;fill-opacity:1;fill-rule:nonzero;stroke:none"
         d="m 2124.82,1048.2 c 0,-57.571 -33.02,-98.2 -87.2,-98.2 -49.11,0 -79.99,27.941 -79.99,78.73 v 270.07 h -106.26 v -289.53 c 0,-94.829 54.17,-156.629 150.69,-156.629 60.95,0 94.82,22.859 124.45,62.64 h 2.54 V 864.488 H 2231.2 V 1298.8 h -106.38 v -250.6" /><path
         id="path20"
         style="fill:#100f0d;fill-opacity:1;fill-rule:nonzero;stroke:none"
         d="m 2726.96,864.289 h 106.27 V 1298.6 H 2726.96 V 864.289" /><path
         id="path22"
         style="fill:#100f0d;fill-opacity:1;fill-rule:nonzero;stroke:none"
         d="m 2726.96,1346.76 h 106.27 v 87.3 h -106.27 v -87.3" /><path
         id="path24"
         style="fill:#100f0d;fill-opacity:1;fill-rule:nonzero;stroke:none"
         d="m 3433.8,1298.6 -63.14,-195.56 c -13.54,-39.79 -25.39,-94.83 -25.39,-94.83 h -2.69 c 0,0 -14.4,55.04 -27.95,94.83 l -71.3,195.56 h -113.6 l 116.84,-309.01 c 24.54,-66.031 33,-93.969 33,-117.68 0,-35.551 -18.62,-59.262 -64.33,-59.262 h -40.64 v -90.589 h 67.72 c 88.05,0 118.62,34.722 151.64,131.23 L 3547.1,1298.6 h -113.3" /><path
         id="path26"
         style="fill:#100f0d;fill-opacity:1;fill-rule:nonzero;stroke:none"
         d="M 3031.22,992.129 V 1222.4 h 67.72 v 76.2 h -67.72 v 135.46 H 2925.68 V 1298.6 h -60.1 v -76.2 h 60.1 V 967.578 c 0,-82.129 61.81,-104.129 117.69,-104.129 45.71,0 58.11,1.692 58.11,1.692 v 83.961 c 0,0 -12.7,-0.153 -27.09,-0.153 -25.4,0 -43.17,11.012 -43.17,43.18" /></g></g></svg>
```

**Comment by ziegenberg** [comment]
I would rather go just for the unity tab and not unity logo plus wordmark. It might be hard to read on small screens and with low resolution.

**Comment by ziegenberg** [comment]
This is already fixed by #597 and can be closed.
